charters/DataProcessingPipelines/charter.md

```diff
@@ -0,0 +1,98 @@
+# [HCA Data Processing Pipelines](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The HCA Data Processing Pipelines are cloud-based and portable analysis pipelines that perform transformative analysis on scientific data in the HCA Data Coordination Platform.
+
+## Objectives
+
+The HCA Data Processing Pipelines team develops all pipelines required by the HCA project for all transformative analyses on scientific data types in the HCA Data Coordination Platform (DCP). Pipelines created to respond to the addition of data types and assays and improvements to pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community. 
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports.  
+__Onboarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP.  
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay.  
```

Comment:
This is an odd one, we call this scoping because this investigation gives us the scope of how much work a pipeline will be. You are right it is an investigation. Here we are defining what amount of work will be needed to meet our generic requirements for pipelines.

Reply:
^^ estimation, not investigation.

> Here we are defining what amount of work will be needed to meet our generic requirements for pipelines.

That was my point.